src/blue.ts

```diff
@@ -129,12 +131,22 @@ export function blueProxyFactory(env: MembraneBroker) {
         }
     }
 
-    function lockShadowTarget(shadowTarget: BlueShadowTarget, originalTarget: BlueProxyTarget) {
-        // copying all own properties into the shadowTarget
-        const targetKeys = ownKeys(originalTarget);
+    function copyRedDescriptorsIntoShadowTarget(shadowTarget: BlueShadowTarget, originalTarget: BlueProxyTarget) {
+        const normalizedRedDescriptors = getOwnPropertyDescriptors(originalTarget);
+        const targetKeys = ownKeys(normalizedRedDescriptors);
+        const blueDescriptors = ObjectCreate(null);
         for (let i = 0, len = targetKeys.length; i < len; i += 1) {
-            copyRedDescriptorIntoShadowTarget(shadowTarget, originalTarget, targetKeys[i]);
```

Comment:
remove copyRedDescriptorIntoShadowTarget

Reply:
`copyRedDescriptorIntoShadowTarget` is still being used in 2 places that also cannot switch to `copyRedDescriptorsIntoShadowTarget`.